internal/pkg/cli/pipeline_status.go

```diff
@@ -109,6 +110,9 @@ func (o *pipelineStatusOpts) Ask() error {
 
 // Execute displays the status of the pipeline.
 func (o *pipelineStatusOpts) Execute() error {
+	if o.pipelineName == "" {
```

Comment:
Is it possible for this to occur? I thought we would be prompting for the pipeline name if not present

Reply:
If it's not present 1. as a flag, 2. in the manifest, or 3. deployed, `askPipelineName` just moves on to `Execute()` and (without this addition) returns 
```
✘ describe status of pipeline: get pipeline status: get pipeline state : InvalidParameter: 1 validation error(s) found.
- minimum field size of 1, GetPipelineStateInput.Name.
```
We never prompt for the user to type in the pipeline name, we only provide a `SelectOne` menu with the deployed pipelines as options.

Reply:
Should we instead error out at line 174 when we can't find any deployed pipeline, if there is no other concern?

I think the case where there is no pipeline to be shown may be better handled in `pipeline_show.go` instead of describer. Because that'd be more of a UX thing as opposed to a describe thing.

Reply:
Ah, sorry if my comment above was misleading. As is, there is no error message if someone tries to show the status of a nonexistent pipeline; they just get some logged info and copilot exits:
```
% copilot pipeline status --app chuckles
No pipeline manifest in workspace for application chuckles, looking for deployed pipelines.
No pipelines found for application chuckles.
```
As is, line 113 is serving as a safety net before the `Describe()` call to prevent that error message.

Reply:
Update: after commit a97a188, no pipelines yields: 
```
% copilot pipeline status --app chuckles
No pipeline manifest in workspace for application chuckles, looking for deployed pipelines.
✘ no pipelines found for application chuckles
```